package.json

```diff
@@ -36,7 +36,7 @@
     "lint": "run-scripts lint:eslint lint:markup",
     "lint:markup": "prettier --check --end-of-line=auto ./**/*.md ./**/*.json ./**/*.yml",
     "lint:markup:fix": "npm run lint:markup -- --write",
-    "lint:eslint": "eslint ./*.js tests/**/*.js",
+    "lint:eslint": "eslint ./*.js 'tests/**/*.js'",
```

Comment:
Adding the single quotes did solve my problem on macOS but seems to not work on Windows. Argh!!